src/views/product-tutorials-view/components/product-view-content/components/tutorials-stack/types.ts

```diff
@@ -0,0 +1,74 @@
+import { Collection, ProductOption } from 'lib/learn-client/types'
+
+/**
+ * TODO
+ * Many of these types were dropped in from the existing front-end repo.
+ * Need to update to use the analogous types from lib/learn-client/types.
+ * Intent is to make those updates during implementation of TutorialsStack
+ */
+
+interface BaseTutorial {
+  id?: string
+  slug: string
+  name: string
+  description: string
+  readTime: number
+}
+
+type CollectionReference = Pick<
+  Collection,
+  'id' | 'name' | 'shortName' | 'slug' | 'theme' | 'level' | 'ordered'
+>
+
+type DefaultCollection = Omit<CollectionReference, 'level' | 'ordered'>
+
+enum VideoHostOption {
```

Comment:
A couple of these enums already exist in learn-client/types - I'm sure you saw after reading your comment. Just noting in case you want to handle those duplicates on this PR.

Reply:
✅ ah sweet, I was a bit lazy here on the first pass - have resolved some duplicates here, will take another pass in an upcoming `TutorialStackItem` PR 👍

Reply:
✅ ah sweet, I was a bit lazy here on the first pass - have resolved some duplicates here, will take another pass in an upcoming `TutorialStackItem` PR 👍